.tx/config

```diff
@@ -3,8 +3,8 @@ host     = https://www.transifex.com
 lang_map = id: in, ja_JP: ja, nl_NL: nl, pt_BR: pt-rBR, zh_CN: zh-rCN, zh_TW: zh-rTW
 
 [o:delta-chat:p:delta-chat-app:r:stringsxml]
-file_filter = res/values-<lang>/strings.xml
-source_file = res/values/strings.xml
+file_filter = src/main/res/values-<lang>/strings.xml
+source_file = src/main/res/values/strings.xml
```

Comment:
I am not sure if this will make the tx program to output to src/main/res instead of ./res, this needs to be tested

Reply:
should be fine, we will find out easily sooner or later :)